ng5/src/app/screens/auth/screens/book-detail/book-detail.component.scss

```diff
@@ -0,0 +1,51 @@
+@import '../../../../../sass/colors.scss';
+
+.book-container {
+  background-color: $white;
+  box-shadow: 5px 7px 5px 0px $grey-secundary;
+  display: flex;
+  justify-content: center;
+  margin: 200px auto;
+
+  .book-logo {
+    height: 300px;
+    margin: 20px;
+    width: 200px;
```

Comment:
by setting the height and width you can change the aspect ratio of the image. you can use object fit property to manage that